mlir/tools/rocmlir-tuning-driver/CMakeLists.txt

```diff
@@ -65,5 +65,9 @@ set_property(TARGET rocmlir-tuning-driver
 
   llvm_update_compile_flags(rocmlir-tuning-driver)
   target_link_libraries(rocmlir-tuning-driver PRIVATE ${LIBS})
+  target_link_libraries(rocmlir-tuning-driver PUBLIC benchmark-driver-utils)
   target_link_libraries(rocmlir-tuning-driver PUBLIC hip::host hip::amdhip64)
+  target_include_directories(rocmlir-tuning-driver PUBLIC
+   "$<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}>"
```

Comment:
@sjw36 Will this break the submodule stuff?

I think we might want to stick with a relative path for now

Reply:
I am happy both ways, but what is the "submodule stuff"?

Reply:
If someone makes rocMLIR a submodule, then all our CMake files need to use CMAKE_CURRENT_<SOURCE/BINARY>_DIR. Please update this one.